content/docs/guide/mapping/hammer_plus_plus_setup.md

```diff
@@ -7,43 +7,77 @@ tags:
   - tools
 ---
 
-Momentum Mod is not officially supported by Hammer++, but can be made compatible by following this guide.
+## Background
+
+Momentum Mod ships with a customized build of Hammer (named "Strata Hammer") which includes some, but not all of Hammer++'s features. Our development has been geared more towards stability than new features, with a focus on porting to Qt for cross platform UI and bug fixing. While Strata Hammer should be perfectly usable (and we appreciate testers), it will be a while until Hammer++ features like accurate lighting preview are fully implemented. If you prefer using Hammer++, that is completely reasonable and this guide should get you set up.
+
+The Garry's Mod build of Hammer++ is recommended due to it being much easier to set up. The CS:GO build will require more tools and disk space to set up, but supports more recent model and texture formats (https://developer.valvesoftware.com/wiki/Category:Counter-Strike:_Global_Offensive_pixel_shaders).
```

Comment:
Sorry but I still think this can be phrased because - like you said yesterday there doesn't seem to be a difference with model formats, which ficool seemed to confirm in red recently.

How about,
```
The CS:GO build will require more tools and disk space to set up, but is required for [some shaders](https://developer.valvesoftware.com/wiki/Category:Counter-Strike:_Global_Offensive_pixel_shaders), in particular 4WayBlend.
```
Also I wonder if some of the custom shaders for weapon finishes like Pearlescent or whatever it's called will only with on CS:GO, but I think with this phrasing it won't be too surprising if certain other shaders like that don't work either.